modules/openapi-generator/src/main/java/org/openapitools/codegen/languages/CClientCodegen.java

```diff
@@ -0,0 +1,700 @@
+/*
+ * Copyright 2018 OpenAPI-Generator Contributors (https://openapi-generator.tech)
+ * Copyright 2018 SmartBear Software
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.openapitools.codegen.languages;
+
+import io.swagger.v3.oas.models.media.ArraySchema;
+import io.swagger.v3.oas.models.media.Schema;
+import org.apache.commons.lang3.StringUtils;
+import org.openapitools.codegen.*;
+import org.openapitools.codegen.utils.ModelUtils;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.io.File;
+import java.util.Arrays;
+import java.util.Iterator;
+import java.util.Map;
+
+public class CClientCodegen extends DefaultCodegen implements CodegenConfig {
```

Comment:
Wouldn't it be better to name it CCUrlClientCodegen, so that the name reflects the library used as well? maybe in the future we have more than one c client

Reply:
Good idea 👍

Reply:
I've renamed it as "CLibcurlClientCodegen.java" to follow the {language}{framework|library}{Client|Server}Codegen pattern.

Ref: https://github.com/curl/curl